README.md

```diff
@@ -9,7 +9,7 @@ The format is a single ZIP archive that bundles various delimited text files des
 
 ## Format Comparison
 
-The ColDP format was developed to overcome limitations existing in currently used formats for sharing taxonomic information, namely [Darwin Core Archives](https://dwc.tdwg.org/text/) and the Catalogue of Life submission format also known as [ACEF](http://www.catalogueoflife.org/content/contributing-your-data#ACEF) (Annual Checklist Exchange Format). Darwin Core Archives and ACEF could still be used for exchanging data to and from the Catalogue of Life clearinghouse, but the COLDP format will support the most features. The following table provides an overview of different features supported in each of the 3 formats:
+The COLDP format was developed to overcome limitations existing in currently used formats for sharing taxonomic information, namely [Darwin Core Archives](https://dwc.tdwg.org/text/) and the Catalogue of Life submission format also known as [ACEF](http://www.catalogueoflife.org/content/contributing-your-data#ACEF) (Annual Checklist Exchange Format). Darwin Core Archives and ACEF could still be used for exchanging data to and from the Catalogue of Life ChecklistBank, but the COLDP format will support the most features. The following table provides an overview of different features supported in each of the 3 formats:
```

Comment:
I did not use COLDP on purpose as it sounds like COLD-P. Can we please stick with ColDP? It's been circulated at TDWG, in presentations, its in our code and separates COL from DataPackage much cleaner

Reply:
Fine, leave it.